<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>447</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Калькулятор поиска корней </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-image: url(E:/QTT/Tvorcheskaya/vspomog/fon.jpg);

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   //border:  3px solid yellow;
color: red;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>               0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>60</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
   //border: 3px solid yellow;
color: red;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>               0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>392</x>
      <y>60</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  // border: 3px solid yellow;
color: red;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>               0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>60</y>
      <width>118</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  // border: 3px solid yellow;
color: red;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>             0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="four">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>264</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tree">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>327</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="six">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>263</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eith">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="one">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>327</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="two">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>327</y>
      <width>63</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="seven">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="five">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>265</y>
      <width>63</width>
      <height>63</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nine">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>201</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zerro">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
   border: 1px solid gray;
 background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="otvet_1">
    <property name="geometry">
     <rect>
      <x>432</x>
      <y>138</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>Ответ</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="zpt">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plmin">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>390</y>
      <width>65</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>138</y>
      <width>421</width>
      <height>315</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';

  border: 3px solid black;
 coloir:red;
}


background-color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AC">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>138</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid gray;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Blabel">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>55</y>
      <width>132</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}

/*QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);  
}*/</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Blabel_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>118</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: transparent;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Blabel_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Blabel_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>130</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background: transparent;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Diskkrim">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="D_1">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>210</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="koren">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="K_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="koren_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="K_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>250</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rezult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>351</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:red;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ukaz">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color: red;
}</string>
    </property>
    <property name="text">
     <string>                Выберете поле для ввода коффициентов</string>
    </property>
   </widget>
   <zorder>label_8</zorder>
   <zorder>K_1</zorder>
   <zorder>koren_2</zorder>
   <zorder>Diskkrim</zorder>
   <zorder>koren</zorder>
   <zorder>K_2</zorder>
   <zorder>D_1</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>Blabel_3</zorder>
   <zorder>Blabel_2</zorder>
   <zorder>Blabel_4</zorder>
   <zorder>four</zorder>
   <zorder>tree</zorder>
   <zorder>six</zorder>
   <zorder>eith</zorder>
   <zorder>one</zorder>
   <zorder>two</zorder>
   <zorder>seven</zorder>
   <zorder>five</zorder>
   <zorder>nine</zorder>
   <zorder>zerro</zorder>
   <zorder>otvet_1</zorder>
   <zorder>zpt</zorder>
   <zorder>plmin</zorder>
   <zorder>AC</zorder>
   <zorder>rezult</zorder>
   <zorder>Blabel</zorder>
   <zorder>ukaz</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
